src/app/products/web3storage/page.tsx

```diff
@@ -0,0 +1,78 @@
+import { Header } from "@/components/nav"
+import { Metadata } from "next"
+import Image from "next/image"
+import Ring from '@/../public/images/illustrations/ring.png'
+
+export const metadata: Metadata = {
+  title: 'Features'
+}
+
+export default async function FeaturesPage () {
+  return (
+    <div>
+      <div className="bg-grad">
+        <Header />
+        <section className='max-w-6xl mx-auto pt-20 md:pt-28 px-3 pb-8'>
+          <div className="md:flex flex-row-reverse">
+            <div className="md:w-2/3">
+              <h1 className="font-bold text-4xl text-zinc-950">What is web3.storage</h1>
+              <p className="leading-relaxed pt-8 max-w-3xl text-xl text-zinc-950">At the core of web3.storage is a storage service to safely secure and make your data available - giving developers the power of decentralized storage and content addressing via simple client libraries or an HTTP API.</p>
```

Comment:
```suggestion
              <p className="leading-relaxed pt-8 max-w-3xl text-xl text-zinc-950">At the core of web3.storage is a storage service to safely secure and make your data available - giving developers the power of decentralized storage and content addressing via simple client libraries.</p>
```

We do not support HTTP at the moment afaik, or I mean if they also handle the UCANs